tests/generate/gitea/knope.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+
+[[workflows]]
+name = "release"
+
+[[workflows.steps]]
+type = "PrepareRelease"
+
+[[workflows.steps]]
+type = "Command"
+command = "git commit -m \"chore: prepare release $version\" && git push"
+
+[workflows.steps.variables]
+"$version" = "Version"
+
+[[workflows.steps]]
+type = "Release"
+
+[[workflows]]
+name = "document-change"
+
+[[workflows.steps]]
+type = "CreateChangeFile"
+
+[gitea]
+owner = "knope-dev"
+repo = "knope"
+host = "https://codeberg.org"
```

Comment:
Should the host have the `https://`? I thought that was being added in code. The examples in the docs do not have a scheme.

Reply:
yeah i forgot to update the doc examples. https is only assumed with known public instances. A private/internal instance however doesn't have to be https. Not everyone bothers using HTTPS within their own network after all. ^^